### PR TITLE
docs(webhosting): migrate webhosting

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+meta:
+  title: How to migrate a cPanel webhosting package to Scaleway
```
Comment (nerda-codes) [inline managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx:3]:
```suggestion
  title: How to migrate a cPanel web hosting package to Scaleway
```

File: managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  description: This page explains how to migrate a cPanel webhosting package to Scaleway.
```
Comment (nerda-codes) [inline managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx:4]:
```suggestion
  description: This page explains how to migrate a cPanel web hosting package to Scaleway.
```

File: managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+content:
+  h1: How to migrate a cPanel webhosting package to Scaleway
```
Comment (nerda-codes) [inline managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx:6]:
```suggestion
  h1: How to migrate a cPanel web hosting package to Scaleway
```

File: managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  paragraph: This page explains how to migrate a cPanel webhosting package to Scaleway.
```
Comment (nerda-codes) [inline managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx:7]:
```suggestion
  paragraph: This page explains how to migrate a cPanel web hosting package to Scaleway.
```

File: managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+</Message>
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a [Web Hosting Cloud (cPanel)](/dedibox-console/cpanel-hosting/quickstart/) plan in the Dedibox console
+</Message>
+
+1. Go to the [list of web hosting accounts](https://console.online.net/fr/mut/list) and select the web hosting you want to migrte.
```
Comment (nerda-codes) [inline managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx:30]:
```suggestion
1. Go to the [list of web hosting accounts](https://console.online.net/fr/mut/list) and select the web hosting you want to migrate.
```

File: managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+2. Click **Migrate** on the webhosting overwiew page. A pop-up displays.
+3. Review the information regarding your web hosting and click **Migrate** to pre-fill the migration information in the Scaleway console. You are being redirected to the Scaleway console.
+4. The migration information for your webhosting display. Verify that you are in the [destination project](/console/my-project/quickstart/) for your hosting. Then click **Migrate your Web Hosting**.
```
Comment (nerda-codes) [inline managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx:33]:
```suggestion
4. The migration information for your web hosting display. Verify that you are in the [destination project](/console/my-project/quickstart/) for your hosting. Then click **Migrate your Web Hosting**.
```

**Review by jsudraud — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

File: managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+<Message type="tip">
+  We highly recommend utilizing our name servers for a seamless migration experience:
+    * `ns1.whm.fr-par.scw.cloud.`
+    * `ns2.whm.fr-par.scw.cloud.`
+  
+  If you are managing your DNS zone by yourself, you can locate all the necessary DNS pointers for configuration within the **DNS verification** tab of your Web Hosting in the Scaleway console.
+</Message>
+
+1. Go to the [list of web hosting accounts](https://console.online.net/en/mut/list) and select the web hosting you want to migrate.
+2. Click **Migrate** on the web hosting overwiew. This action will trigger a pop-up to appear.
```
Comment (mardoj) [inline managed-services/webhosting/how-to/migrate-webhosting-to-scaleway.mdx:40]: overview

**Pull request merged**